src/extension.ts

```diff
@@ -112,6 +112,12 @@ async function openTemplateAndSaveNewFile(type: string, namespace: string, filen
 
             editor.selection = newselection;
         }
+
+        const includeNamespaces = vscode.workspace.getConfiguration().get('csharpextensions.includeNamespaces', true);
+
+        if (includeNamespaces) {
+            text = text.replace('${namespaces}', `using System;${EOL}using System.Collections.Generic;${EOL}using System.Linq;${EOL}using System.Threading.Tasks;`)
```

Comment:
I didn't tested but I think the `namespaces` is in plural form here but in the `.tmpl` files not. And if I'm correct this will break
- apicontroller.tmpl
- and controller.tmpl

because of the missing ASP namespace(s).